documentation/assemblies/tracing/assembly-instrumenting-kafka-clients-tracers.adoc

```diff
@@ -0,0 +1,11 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-tracing.adoc
+[id='assembly-instrumenting-kafka-clients-tracers-{context}']
+= Instrumenting Kafka clients with tracers
+
+Instrument Kafka Producer, Consumer, and Streams API applications for distributed tracing.
```

Comment:
Are we capitalising producer and consumer? I haven't been! Would it be better saying _Kafka producer and consumer clients, and..._

Reply:
Hmm, as part of this PR I removed the caps from Producer and Consumer throughout the section. I agree we should not capitalise.